ProjectLighthouse.Servers.GameServer/Controllers/MessageController.cs

```diff
@@ -120,7 +120,7 @@ public async Task<IActionResult> Filter()
 
         string username = await this.database.UsernameFromGameToken(token);
 
-        Logger.Info($"{username}: {message} / {scannedText}", LogArea.Filter);
+        if (ServerConfiguration.Instance.LogChatFiltering) Logger.Info($"{username}: {message} / {scannedText}", LogArea.Filter);
```

Comment:
```suggestion
        if (ServerConfiguration.Instance.LogChatFiltering) 
          Logger.Info($"{username}: {message} / {scannedText}", LogArea.Filter);
```
This should probably be split into 2 separate lines because it's over 80 characters long